client/my-sites/site-settings/jetpack-module-toggle.jsx

```diff
@@ -31,7 +31,7 @@ class JetpackModuleToggle extends Component {
 	};
 
 	static propTypes = {
-		siteId: PropTypes.number.isRequired,
```

Comment:
What is the rationale behind this change? I think this component does not function properly without a `siteId, so I'd suggest keeping this one required, but instead, altering the logic either in this component or in the parent component to not render the toggle if `siteId` is `null`. Or is there other specific reason to make it non-required?